SQL/0000-00-00-schema.sql

```diff
@@ -1496,6 +1496,8 @@ INSERT INTO parameter_type (Name, Type, Description, RangeMin, RangeMax, SourceF
   ('Visit_label','varchar(255)','Visit_label',null,null,'visit_label','session',null,1,null),
   ('candidate_dob','date','Candidate_Dob',null,null,'DoB','candidate',null,1,null);
 
+INSERT INTO parameter_type (Name, Type) Values ("DeepQC", "double");
```

Comment:
Hi @llevitis 

I am not sure you even need this mysql patch.
On the MRI side, the utilities do this automatically for you...
am I not right @cmadjar?

Reply:
@llevitis @MounaSafiHarab I agree with Mouna. This will automatically be done by the imaging pipeline. No need to create a patch here.